Composer/packages/client/src/recoilModel/atoms/appState.ts

```diff
@@ -370,6 +371,11 @@ export const surveyEligibilityState = atom<boolean>({
   default: false,
 });
 
+export const machineInfoState = atom<MachineInfo>({
+  key: getFullyQualifiedKey('machineInfoState'),
+  default: { id: '', os: '' },
```

Comment:
Nit: I think id and os should default to undefined or null, so it's clear they haven't been set yet.